src/content/docs/en/technology/chain/rollup.mdx

```diff
@@ -141,7 +142,6 @@ This section describes the codec of three data structures in the Rollup contract
 | `block[i]`       | 60      | `BlockContext` | `60*i+1`  | The block information of `i+1`-th block                                                                                                           |
 | ...              | ...     | ...            | ...       | ...                                                                                                                                               |
 | `block[n-1]`     | 60      | `BlockContext` | `60*n-59` | The block information of the last block                                                                                                           |
-| `l2Transactions` | dynamic | `bytes`        | `60*n+1`  | The concatenated RLP encoding of L2 transactions with signatures. The byte length (`uint32`) of RLP encoding is inserted before each transaction. |
```

Comment:
Do we need a footnote to denote items that are different between testnet and mainnet here?